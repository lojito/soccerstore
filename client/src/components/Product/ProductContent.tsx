import { type FC, type ReactNode } from "react";
import { styled } from "styled-components";

const DivWrapper = styled.div`
  color: #2d2f31;
  font-weight: bold;
  font-size: 1.6rem;
  margin-bottom: 5px;
  text-align: left;
  width: 225px;
  margin: 0 auto;
`;

type Props = {
  children: ReactNode;
};

const ProductContent: FC<Props> = ({ children }) => {
  return <DivWrapper>{children}</DivWrapper>;
};

export default ProductContent;
