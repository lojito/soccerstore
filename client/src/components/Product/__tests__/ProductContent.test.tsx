import { render } from "@testing-library/react";
import ProductContent from "../ProductContent";

describe("ProductContent", () => {
  it("should render correctly", () => {
    const { asFragment } = render(
      <ProductContent children="Balon Blanco y Negro" />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
