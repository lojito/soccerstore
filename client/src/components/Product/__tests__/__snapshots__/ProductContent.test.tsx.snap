// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProductContent > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-beySPh fUhQlh"
  >
    Balon Blanco y Negro
  </div>
</DocumentFragment>
`;
